players/g1_player.py

```diff
@@ -235,6 +235,9 @@ def __init__(self, skill: int, rng: np.random.Generator, logger: logging.Logger,
         max_dist = 200 + self.skill
         self.max_ddist = scipy_stats.norm(max_dist, max_dist / self.skill)
 
+        max_sandtrap_dist = max_dist/2
+        self.max_sandtrap_ddist = scipy_stats.norm(max_sandtrap_dist, max_dist / self.skill)
```

Comment:
> When hitting from a sand trap, *the distance rating is halved*, and the *standard deviations for the angle and distance distributions are doubled*.

Should the distribution be like the following instead?

```python
scipy_stats.norm(max_sandtrap_dist, 2 * max_dist / self.skill)
```

Reply:
max_dist is being used to calculate the variance which is equal to max_sandtrap_dist*2 (so it is doubled) , just reused variable for computation time